modules/zenith-public/lua/2-base/abilities.lua

```diff
@@ -0,0 +1,160 @@
+require('modules/module_utils')
+
+local m = Module:new('abilities')
+
+local effectAdjustments =
+{
+    -- Warrior
+    blood_rage =
+    {
+        effectId = xi.effect.BLOOD_RAGE,
+        add =
+        {
+            power = -10, -- Changed from 20% to 10%
+        },
+    },
+    -- Monk
+    impetus =
+    {
+        effectId = xi.effect.IMPETUS,
+        add =
+        {
+            duration = -60000, -- Changed from 180s to 120s
+        },
+    },
+    -- Thief
+    conspirator =
+    {
+        effectId = xi.effect.CONSPIRATOR,
+        multiply =
+        {
+            power = 0.75, -- 25% reduction
+            subpower = 0.75, -- 25% reduction
+        },
+    },
+    sepulcher =
+    {
+        effectId = xi.effect.SEPULCHER,
+        add =
+        {
+            duration = -60000, -- Changed from 180s to 120s
+        },
+    },
+    -- Samurai
+    hagakure =
+    {
+        effectId = xi.effect.HAGAKURE,
+        multiply =
+        {
+            power = 0.5, -- 50% reduction
+            subpower = 0.5, -- 50% reduction
+        },
+    },
+    hamanoha =
+    {
+        effectId = xi.effect.HAMANOHA,
+        add =
+        {
+            duration = -60000, -- Changed from 180s to 120s
+        },
+    },
+    sengikori =
+    {
+        effectId = xi.effect.SENGIKORI,
+        add =
+        {
+            power = -10, -- Changed from 25% to 15%
```

Comment:
```suggestion
            power = -15, -- Changed from 25% to 10%
```
Everything here looks great but I want to reduce this by 5% so the Empyrean feet bonus brings it back to 15%. 20% as a unique multiplier feels a bit strong.

Thank you for your work.